src/covidify/pipeline.sh

```diff
@@ -1,18 +1,23 @@
-#!/bin/bash
+#!/bin/sh
+
+set -e
+
 function banner {
     echo "###"
     echo "### $1"
     echo "###"
 }
 
-ENV=$1
-OUT_FDR=$2
+ENV="$(dirname "$(realpath "$0")")" # current directory
+OUT_FDR="${2:-/tmp/data/}"
```

Comment:
Few things here:
- ENV isn't the current directory, its the location where the python package is found in `site-packages`. For example, you want to run `data_prep.py` but you are in the  desktop directory, it can't find it. This ensure you are running the installed `data_prep` and not the one in the repo

- So the reason why OUT_FDR is passed is because i want people to be able to choose where they want the reports/data to be located. so you can run a command like

  - `covidify run --output=/Users/username/Documents/folder`
  - If output isn't specified then default to desktop. 

- It would also be a good idea for the source to be a option as well in CLI,  so you could do something like
  - `covidify run --output=/Users/username/Documents/folder --source=wikipedia` and default to github if the flag is not specified.